src/test/java/seedu/fma/testutil/LogBookBuilder.java

```diff
@@ -0,0 +1,34 @@
+package seedu.fma.testutil;
+
+import seedu.fma.model.LogBook;
+import seedu.fma.model.log.Log;
+
+/**
+ * A utility class to help with building LogBook objects.
+ * Example usage: <br>
+ *     {@code LogBook ab = new LogBookBuilder().withLog("John", "Doe").build();}
```

Comment:
Shouldn't the withLog() method take in a Log object as an argument instead of 2 strings?

Reply:
ooh good catch